src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -50,6 +51,7 @@ export interface ShippingState {
   address: Address
   shippingOption: OrderFulfillmentType
   isComittingMutation: boolean
+  isModalOpen: boolean
```

Comment:
This state variable is a bit generic. It's tied to the behavior of the Error modal but that really doesn't show based on the name. Maybe `isErrorModalOpen` or something?